generated/nirfmxlte/nirfmxlte.proto

```diff
@@ -13,43 +13,5072 @@ option csharp_namespace = "NationalInstruments.Grpc.NiRFmxLTE";
+  rpc WaitForAcquisitionComplete(WaitForAcquisitionCompleteRequest) returns (WaitForAcquisitionCompleteResponse);
+  rpc WaitForMeasurementComplete(WaitForMeasurementCompleteRequest) returns (WaitForMeasurementCompleteResponse);
 }
 
 enum NiRFmxLTEAttribute {
   NIRFMXLTE_ATTRIBUTE_UNSPECIFIED = 0;
+  NIRFMXLTE_ATTRIBUTE_CENTER_FREQUENCY = 3145729;
+  NIRFMXLTE_ATTRIBUTE_REFERENCE_LEVEL = 3145730;
+  NIRFMXLTE_ATTRIBUTE_EXTERNAL_ATTENUATION = 3145731;
+  NIRFMXLTE_ATTRIBUTE_TRIGGER_TYPE = 3145732;
+  NIRFMXLTE_ATTRIBUTE_DIGITAL_EDGE_TRIGGER_SOURCE = 3145733;
+  NIRFMXLTE_ATTRIBUTE_DIGITAL_EDGE_TRIGGER_EDGE = 3145734;
+  NIRFMXLTE_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_SOURCE = 3145735;
+  NIRFMXLTE_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_LEVEL = 3145736;
+  NIRFMXLTE_ATTRIBUTE_IQ_POWER_EDGE_TRIGGER_SLOPE = 3145737;
+  NIRFMXLTE_ATTRIBUTE_TRIGGER_DELAY = 3145738;
+  NIRFMXLTE_ATTRIBUTE_TRIGGER_MINIMUM_QUIET_TIME_MODE = 3145739;
+  NIRFMXLTE_ATTRIBUTE_TRIGGER_MINIMUM_QUIET_TIME_DURATION = 3145740;
+  NIRFMXLTE_ATTRIBUTE_DUPLEX_SCHEME = 3145741;
+  NIRFMXLTE_ATTRIBUTE_UPLINK_DOWNLINK_CONFIGURATION = 3145742;
+  NIRFMXLTE_ATTRIBUTE_NUMBER_OF_COMPONENT_CARRIERS = 3145743;
+  NIRFMXLTE_ATTRIBUTE_COMPONENT_CARRIER_BANDWIDTH = 3145744;
+  NIRFMXLTE_ATTRIBUTE_COMPONENT_CARRIER_FREQUENCY = 3145745;
+  NIRFMXLTE_ATTRIBUTE_CELL_ID = 3145746;
+  NIRFMXLTE_ATTRIBUTE_COMPONENT_CARRIER_SPACING_TYPE = 3145747;
+  NIRFMXLTE_ATTRIBUTE_COMPONENT_CARRIER_AT_CENTER_FREQUENCY = 3145748;
+  NIRFMXLTE_ATTRIBUTE_CYCLIC_PREFIX_MODE = 3145749;
+  NIRFMXLTE_ATTRIBUTE_BAND = 3145751;
+  NIRFMXLTE_ATTRIBUTE_UPLINK_GROUP_HOPPING_ENABLED = 3145753;
+  NIRFMXLTE_ATTRIBUTE_UPLINK_SEQUENCE_HOPPING_ENABLED = 3145754;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_MODULATION_TYPE = 3145755;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_NUMBER_OF_RESOURCE_BLOCK_CLUSTERS = 3145756;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_RESOURCE_BLOCK_OFFSET = 3145758;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_N_DMRS_1 = 3145759;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_N_DMRS_2 = 3145760;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_DELTA_SEQUENCE_SHIFT = 3145761;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_NUMBER_OF_RESOURCE_BLOCKS = 3145762;
+  NIRFMXLTE_ATTRIBUTE_NUMBER_OF_SUBBLOCKS = 3145763;
+  NIRFMXLTE_ATTRIBUTE_SUBBLOCK_FREQUENCY_DEFINITION = 3145764;
+  NIRFMXLTE_ATTRIBUTE_AUTO_RESOURCE_BLOCK_DETECTION_ENABLED = 3145766;
+  NIRFMXLTE_ATTRIBUTE_PUSCH_POWER = 3145767;
+  NIRFMXLTE_ATTRIBUTE_AUTO_DMRS_DETECTION_ENABLED = 3145768;
+  NIRFMXLTE_ATTRIBUTE_LINK_DIRECTION = 3145769;
+  NIRFMXLTE_ATTRIBUTE_SPECIAL_SUBFRAME_CONFIGURATION = 3145770;
+  NIRFMXLTE_ATTRIBUTE_NUMBER_OF_DUT_ANTENNAS = 3145771;
+  NIRFMXLTE_ATTRIBUTE_TRANSMIT_ANTENNA_TO_ANALYZE = 3145772;
+  NIRFMXLTE_ATTRIBUTE_SRS_ENABLED = 3145773;
+  NIRFMXLTE_ATTRIBUTE_SRS_SUBFRAME_CONFIGURATION = 3145774;
+  NIRFMXLTE_ATTRIBUTE_SRS_C_SRS = 3145775;
+  NIRFMXLTE_ATTRIBUTE_SRS_B_SRS = 3145776;
+  NIRFMXLTE_ATTRIBUTE_SRS_I_SRS = 3145777;
+  NIRFMXLTE_ATTRIBUTE_SRS_n_RRC = 3145778;
```

Comment:
These mixed case attributes snuck through. We need `UPPER_SNAKE` for these per [proto style guidelines](https://developers.google.com/protocol-buffers/docs/style#enums).